clinica/pipelines/t1_freesurfer/t1_freesurfer_cli.py

```diff
@@ -22,36 +23,45 @@ def define_options(self):
                                   help='Path to the BIDS directory.')
         clinica_comp.add_argument("caps_directory",
                                   help='Path to the CAPS directory.')
+
         # Optional arguments (e.g. FWHM)
         optional = self._args.add_argument_group(PIPELINE_CATEGORIES['OPTIONAL'])
         optional.add_argument("-raa", "--recon_all_args",
-                              help='Additional flags for recon-all command line (default: --recon_all_args "-qcache")')
+                              metavar='flag(s)', type=str, default="-qcache",
+                              help='Additional flags for recon-all command line '
+                                   '(default: --recon_all_args="%(default)s")')
+
         # Clinica standard arguments (e.g. --n_procs)
         clinica_opt = self._args.add_argument_group(PIPELINE_CATEGORIES['CLINICA_OPTIONAL'])
         clinica_opt.add_argument("-tsv", "--subjects_sessions_tsv",
                                  help='TSV file containing a list of subjects with their sessions.')
         clinica_opt.add_argument("-wd", "--working_directory",
                                  help='Temporary directory to store pipelines intermediate results')
         clinica_opt.add_argument("-np", "--n_procs",
-                                 metavar=('N'), type=int,
+                                 metavar='N', type=int,
                                  help='Number of cores used to run in parallel')
 
     def run_command(self, args):
         """Run the pipeline with defined args."""
         import os
-        import datetime
         from colorama import Fore
+        from tempfile import mkdtemp
+        from .t1_freesurfer_pipeline import T1FreeSurfer
         from clinica.utils.exceptions import ClinicaException
         from clinica.utils.stream import cprint
-        from .t1_freesurfer_pipeline import T1FreeSurfer
-        from tempfile import mkdtemp
+        from clinica.utils.ux import print_end_pipeline
 
         pipeline = T1FreeSurfer(
             bids_directory=self.absolute_path(args.bids_directory),
             caps_directory=self.absolute_path(args.caps_directory),
             tsv_file=self.absolute_path(args.subjects_sessions_tsv)
         )
 
+        if "-dontrun" in args.recon_all_args.split(' '):
```

Comment:
Currenly, execution of `T1FreeSurfer` is disabled due to very long regression tests.

When I investigated how Clinica was covered by our tests, I found out that instanciation tests covered almost everything in `T1FreeSurfer` class in `t1_freesurfer_pipeline.py` but `t1_freesurfer_utils.py` was not covered.

Since `T1FreeSurfer` simply calls `recon-all` and creates TSV files, my idea is to
- add `-dontrun` for `test_run_t1_freesurfer`
- add unit tests to `generate_regional_measures` (in `utils/freesurfer.py`) i.e. the function that creates TSV files
- add unit tests to `save_to_caps` (in `t1_freesurfer_utils.py`).

By doing so, it would decrease significantly computational time (few minutes vs 8-10 hours) while covering completely `t1_freesurfer` and critical functions.

Reply:
Ok ! Good job !